apps/explorer/vercel.json

```diff
@@ -0,0 +1,11 @@
+{
+  "rewrites": [
+    {
+      "source": "/(.*)",
+      "destination": "/index.html"
+    }
+  ],
```

Comment:
The main part of the fix: rewrite all requests to index.html so it's treated as a SPA.